README.Rmd

```diff
@@ -21,248 +21,58 @@ knitr::opts_chunk$set(
-- `odbc.ini` - which defines connection options
-
-The DSN configuration files can be defined globally for all users of the
-system, often at
-`/etc/odbc.ini`, `/opt/local/etc/odbc.ini` or `/opt/homebrew/etc/odbc.ini`, the
-exact location depends on what option was used when compiling unixODBC. `odbcinst -j`
-can be used to find the exact location. Alternatively the `ODBCSYSINI` environment
-variable can be used to specify the location of the configuration files. Ex. `ODBCSYSINI=~/ODBC`
-
-A local DSN file can also be used with the files `~/.odbc.ini` and `~/.odbcinst.ini`.
-
-##### MacOS aarch64
-If ODBC has trouble locating your system data source names, you may need to
-override the default location where ODBC looks for your configuration files.
-You can use either of the options below to specify the location of your DSN
-configuration files.
-
-###### Option 1: Save Setting to `~/.Renviron`
-1. Create or open the `~/.Renviron` file.
-2. Add `ODBCSYSINI=/opt/homebrew/etc` to your `~/.Renviron` file and
-   save your changes.
-3. Restart any open R sessions before connecting to a database.
-
-###### Option 2: Set `ODBCSYSINI` Environment Variable
-1. Set the `ODBCSYSINI` environment variable, eg. `ODBCSYSINI=/opt/homebrew/etc`.
-2. Restart any open R sessions before connecting to a database.
-
-##### odbcinst.ini
-Contains driver information, particularly the name of the driver library.
-Multiple drivers can be specified in the same file.
-```ini
-[PostgreSQL Driver]
-Driver          = /usr/local/lib/psqlodbcw.so
-
-[SQLite Driver]
-Driver          = /usr/local/lib/libsqlite3odbc.dylib
+# install.packages("pak")
+pak::pak("r-dbi/odbc")
 ```
 
-On MacOS aarch64 machines, drivers installed via homebrew are in a
-different location, as seen below.
-
-```ini
-[PostgreSQL Driver]
-Driver          = /opt/homebrew/lib/psqlodbcw.so
-
-[SQLite Driver]
-Driver          = /opt/homebrew/lib/libsqlite3odbc.dylib
-```
+odbc and its dependencies use C++11 features. Therefore, you need [gcc 4.8](https://gcc.gnu.org/), [clang 3.3](https://clang.llvm.org), or [Rtools 3.3](https://CRAN.R-project.org/bin/windows/Rtools/) or later.
 
-##### odbc.ini
-Contains connection information, particularly the username, password, database
-and host information. The Driver line corresponds to the driver defined in
-`odbcinst.ini`.
-```ini
-[PostgreSQL]
-Driver              = PostgreSQL Driver
-Database            = test_db
-Servername          = localhost
-UserName            = postgres
-Password            = password
-Port                = 5432
-
-[SQLite]
-Driver              = SQLite Driver
-Database            = /tmp/testing
-```
+## Usage
 
-See also: [unixODBC without the GUI](https://www.unixodbc.org/odbcinst.html) for more information and examples.
+All of the following examples assume you have already created a connection `con`. See the "Installing and Configuring Drivers" vignette for more information on establishing a connection.
```

Comment:
Use `vignette("setup")` to get an automatic link on the pkgdown site?

Reply:
[1f9f52f](https://github.com/r-dbi/odbc/pull/652/commits/1f9f52f2e6cf80ee7f5e91c1126359c1eb5f58b1).